README.Rmd

```diff
@@ -16,11 +16,12 @@ knitr::opts_chunk$set(
 # sfrouting 🏗️ work in progress 🏗️
 
 <!-- badges: start -->
+
 [![R-CMD-check](https://github.com/Robinlovelace/sfrouting/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/Robinlovelace/sfrouting/actions/workflows/R-CMD-check.yaml)
+
 <!-- badges: end -->
 
-The goal of sfrouting is to enable people to generate routes for reproducible research.
-The package provides an interface between {[sfnetworks](https://luukvdmeer.github.io/sfnetworks/)} and {[cppRouting](https://cran.r-project.org/package=cppRouting)} R packages.
+The goal of sfrouting is to enable people to generate routes for reproducible research. The package provides an interface between {[sfnetworks](https://luukvdmeer.github.io/sfnetworks/)} and {[cppRouting](https://cran.r-project.org/package=cppRouting)} R packages.
```

Comment:
I'm a fan of sentence case for visual diffs when editing prose. Paragraph breaks fine also. Adding the following to the yaml header should fix it (does for sure for .qmd files) if editing in visual mode:

```
editor:
  markdown:
    wrap: sentence
```